qualtran/cirq_interop/_cirq_interop.py

```diff
@@ -188,67 +204,97 @@ def cirq_optree_to_cbloq(
         signature = Signature([Register('qubits', 1, shape=(len(all_qubits),))])
         cirq_quregs = {'qubits': np.array(all_qubits).reshape(len(all_qubits), 1)}
 
+    cirq_quregs = {k: np.apply_along_axis(QReg, -1, v) for k, v in cirq_quregs.items()}
```

Comment:
there's a comment above

> "qubits" means cirq qubits | "qvars" means bloq Soquets

does this still apply? update or delete

Reply:
It still applies, but I've deleted the line anyway since there are 3 things at play now (Qubits, _QReg's and Soquets)